src/saml2/assertion.py

```diff
@@ -290,7 +290,8 @@ def compile(restrictions):
             for key, items in _mod.RELEASE.items():
                 alist = [k.lower() for k in items]
                 _only_required = getattr(_mod, "ONLY_REQUIRED", {}).get(key, False)
-                _ec[key] = (alist, _only_required)
+                _no_aggregation = getattr(_mod, "NO_AGGREGATION", {}).get(key, False)
```

Comment:
if `NO_AGGREGATION` becomes a set, then this would look like

```suggestion
                _no_aggregation = key in getattr(_mod, "NO_AGGREGATION", set())
```